environments/prod/find-court-tribunal-service-gov-uk.yml

```
@@ -16,3 +16,12 @@ cname:
   - name: "_7c539238d4da39d0fa4242e862930d02.admin"
     ttl: 300
     record: "CA97F4612709C22E162E35D39F71B6A8.32CBBA599F56C6E44500A1CFFF18F733.d85688fce613669775c5.comodoca.com."
+  - name: "admin"
+    ttl: 300
+    record: "hmcts-prod.azurefd.net"
+ - name: "cdnverify.www"
+    ttl: 300
+    record: "cdnverify.hmcts-find-court-tribunal-shutter-prod.azureedge.net"
```

probably would have admin in the name but not 100% if it fails you should get the error message with what it needs to be:
```suggestion
    record: "cdnverify.hmcts-admin-find-court-tribunal-shutter-prod.azureedge.net"
```